docker-compose.zappdev.yaml

```diff
@@ -75,7 +75,7 @@ services:
       DATADIR: /data
 
   arlocal:
-    image: textury/arlocal
+    image: textury/arlocal:v1.1.21
```

Comment:
@diogosmendonca Note - I decided to add the version declaration explicitly here.